docs/azure/integrations-overview.md

```diff
@@ -40,6 +64,119 @@ The preceding code:
 
 The consuming API project uses the connection string information with no knowledge of how the app host configured it. In "publish" mode, the code adds a new Azure Storage resource—which would be reflected in the [deployment manifest](../deployment/manifest-format.md) accordingly. When in "run" mode, the connection string corresponds to a configuration value visible to the app host. It's assumed that any and all role assignments for the target resource have been configured. This means, you'd likely configure an environment variable or a user secret to store the connection string. The configuration is resolved from the `ConnectionStrings__storage` (or `ConnectionStrings:storage`) configuration key. These configuration values can be viewed when the app runs. For more information, see [Resource details](../fundamentals/dashboard/explore.md#resource-details).
 
+## Mark Azure resources as existing
```

Comment:
@captainsafia - what is your thoughts on moving this above `Add existing Azure resources with connection strings`?

Reply:
I like that with this format we have a segue from the connection strings approach to the bicep one. Although it does make it seem like connection strings are preferred to bicep. We can consider re-arranging it and maybe making the connection string stuff an H3 under existing resources instead of an H2?

Reply:
> We can consider re-arranging it and maybe making the connection string stuff an H3 under existing resources instead of an H2?

I like this idea.

> Although it does make it seem like connection strings are preferred to bicep.

Yes, that was my thinking. I think we should suggest using the new "Existing" feature first, because it works so much better and enables so many more scenarios.

Reply:
How about we restructure/flow of the article to be the following (only looking at H2s and only highlighting changes to H3):

- Naming conventions and APIs for expressing Azure resources
- Add Azure resources
  - Mark Azure resources as existing
  - Add existing Azure resources with connection strings
- Infrastructure as code
- Publishing

Perhaps both the existing APIs bits and the connection string blurbs should be H3s, under the Add Azure resources heading? Thoughts? Admittedly, there are other H3s and H4s that were omitted from here.

Reply:
I like moving the "as existing" and "connection string" stuff lower than the "typical devloper experience" stuff. We should have the normal cases first - when you add a new resource, not use an existing one. I also like having "as existing" above "connection string". So I think this proposal sounds good to me.